responses/de/HassGetState.yaml

```
@@ -85,3 +85,11 @@ responses:
             {%- endfor -%}
           {% endif %}
         {% endif %}
+
+      get_volume: |
+        {% if 'volume_level' in state.attributes %}
+          {% set volume = (state.attributes.volume_level | float(0)) * 100 %}
+          {{ slots.name | capitalize }} ist auf {{ volume | round(0) | int }} % eingestellt
+        {% else %}
+          {{ slots.name | capitalize }} hat keinen Lautstärke-Wert
```

```suggestion
        {% elif state.state == "off" %}
          {{ slots.name | capitalize }} ist im Moment ausgeschaltet
        {% else %}
          {{ slots.name | capitalize }} hat keinen Lautstärke-Wert
```

If the media player is off, there is no `volume_level` attribute, so I think we should let the user know the difference.